exercises/strings/longestWord/longestWord.js

```diff
@@ -9,11 +9,26 @@
 
 function longestWord(string) {
   // This is your job. :)
+  let words = []
+  let endIndex = 0
+  let startIndex = 0
+  for (i in string) {
+    if (string[i] === " " || string[i] === ".") {
+      endIndex = Number(i)
+      words.push(string.slice(startIndex,endIndex))
+      startIndex = Number(endIndex) + 1
+    }
+  }
+  words.sort( function(a,b) {
```

Comment:
If `sort` is within the rules of this function, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split definitely is! I'd just use that.